### DISCUSSION TITLE
Bisiklet yarışçıları sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#264] [Zeka Sorusu] [answered]
Aydın,Burhan ve Cemal bisiklet yarışçısıdır. Herkes kendi bisiklet ve formasını kullanmak yerine, bir arkadaşının bisikletine binip diğerinin formasını giymektedir. Cemal'in formasını giyen bisikletçi Burhan'ın bisikletine binmektedir. Bu durumda Aydın'ın bisikletine kim binmektedir ?

**Comment by ElifNurSoyler** [comment, marked as answer]
Aydın = Burhan bisiklet, Cemal forma
Burhan = Cemal bisiklet, Aydın forma
Cemal = Aydın bisiklet, Burhan forma
Cevap = Cemal
